egs/wsj/s5/steps/nnet3/train_dnn.py

```diff
@@ -450,6 +450,9 @@ def Train(args, run_opts):
     feat_dim = GetFeatDim(args.feat_dir)
     ivector_dim = GetIvectorDim(args.online_ivector_dir)
 
+    shutil.copy(args.online_ivector_dir+'/ivector_extractor_id',
```

Comment:
you might want to use try-pass to ensure this doesn't die if the source extractor did has no ivector_extractor_id (for back compatibility)